src/UCTNode.cpp

```diff
@@ -410,53 +410,18 @@ size_t UCTNode::count_nodes() const {
     return nodecount;
 }
 
-// Use this version if you know the child is directly under the parent.
-UCTNode::node_ptr_t UCTNode::find_new_root(const int move) {
+// Used to find new root in UCTSearch
+UCTNode::node_ptr_t UCTNode::find_child(const int move) {
```

Comment:
This might be better name 'return_child' or 'find_child_destructive' to indicate that it destroys part of the subtree